corehq/apps/translations/models.py

```diff
@@ -21,19 +23,37 @@
 class TranslationMixin(Document):
     translations = DictProperty()
 
-    def set_translation(self, lang, key, value):
-        if lang not in self.translations:
-            self.translations[lang] = {}
-        if value is not None:
-            self.translations[lang][key] = value
-        else:
-            del self.translations[lang][key]
-
     def set_translations(self, lang, translations):
         self.translations[lang] = translations
 
 
-class StandaloneTranslationDoc(TranslationMixin, CouchDocLockableMixIn):
+class SMSTranslations(SyncSQLToCouchMixin, models.Model):
+    domain = models.CharField(max_length=255, unique=True)
+    langs = JSONField(default=list)
+    translations = JSONField(default=dict)
+    couch_id = models.CharField(max_length=126, null=True, db_index=True)
+
+    @property
+    def default_lang(self):
+        if len(self.langs) > 0:
```

Comment:
Four lines to one `return self.langs[0] if self.langs else None`?

Reply:
b488996fa36